#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_fix_distclean.dpatch by  <sascha.ittner@modusoft.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix make distclean problem

@DPATCH@

diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/devices/e1000/Makefile.am etherlabmaster-1.5.2+hg4b0b90/devices/e1000/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/devices/e1000/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/devices/e1000/Makefile.am	2015-05-06 09:58:43.745260774 -0400
@@ -350,6 +350,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/devices/e1000e/Makefile.am etherlabmaster-1.5.2+hg4b0b90/devices/e1000e/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/devices/e1000e/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/devices/e1000e/Makefile.am	2015-05-06 09:58:43.749260774 -0400
@@ -428,6 +428,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/devices/Makefile.am etherlabmaster-1.5.2+hg4b0b90/devices/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/devices/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/devices/Makefile.am	2015-05-06 09:58:43.753260774 -0400
@@ -186,6 +186,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/examples/dc_rtai/Makefile.am etherlabmaster-1.5.2+hg4b0b90/examples/dc_rtai/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/examples/dc_rtai/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/examples/dc_rtai/Makefile.am	2015-05-06 09:58:43.753260774 -0400
@@ -44,6 +44,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/examples/mini/Makefile.am etherlabmaster-1.5.2+hg4b0b90/examples/mini/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/examples/mini/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/examples/mini/Makefile.am	2015-05-06 09:58:43.753260774 -0400
@@ -46,6 +46,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/examples/rtai/Makefile.am etherlabmaster-1.5.2+hg4b0b90/examples/rtai/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/examples/rtai/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/examples/rtai/Makefile.am	2015-05-06 09:58:43.757260774 -0400
@@ -44,6 +44,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/examples/tty/Makefile.am etherlabmaster-1.5.2+hg4b0b90/examples/tty/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/examples/tty/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/examples/tty/Makefile.am	2015-05-06 09:58:43.757260774 -0400
@@ -50,6 +50,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/Makefile.am etherlabmaster-1.5.2+hg4b0b90/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/Makefile.am	2015-05-06 09:58:43.757260774 -0400
@@ -88,7 +88,7 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 mydist:
 	hg log --style=changelog $(srcdir) > ChangeLog
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/master/Makefile.am etherlabmaster-1.5.2+hg4b0b90/master/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/master/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/master/Makefile.am	2015-05-06 09:58:43.757260774 -0400
@@ -88,6 +88,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
diff -urNp etherlabmaster-1.5.2+hg4b0b90.orig/tty/Makefile.am etherlabmaster-1.5.2+hg4b0b90/tty/Makefile.am
--- etherlabmaster-1.5.2+hg4b0b90.orig/tty/Makefile.am	2015-02-19 09:19:29.000000000 -0500
+++ etherlabmaster-1.5.2+hg4b0b90/tty/Makefile.am	2015-05-06 09:58:43.761260774 -0400
@@ -46,6 +46,6 @@ modules_install:
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
